source/Firmata/UwpFirmata.cpp

```diff
@@ -45,11 +44,12 @@ namespace {
 
 UwpFirmata::UwpFirmata(
     void
-):
+) :
     _data_buffer(new uint8_t[31]),
```

Comment:
couple things:
* use const for the buffer size
*  since it is const, you can allocate space in the class instead of using a separate heap block